src/hmrc-design-patterns/__tests__/hmrc-design-patterns.integration.js

```diff
@@ -23,56 +23,11 @@ describe('HMRC design patterns', () => {
 
   it('should have the correct meta title', async () => {
     const title = await page.title()
-    expect(title).toBe('HMRC design patterns - Design resources for HMRC – GOV.UK')
+    expect(title).toBe('HMRC design patterns - Design resources for HMRC — GOV.UK')
   })
 
   it('should have the correct page heading', async () => {
     const heading = await page.$eval('h1', el => el.textContent);
     expect(heading).toBe('HMRC design patterns')
   })
-
-  describe('Sub navigation', () => {
```

Comment:
converted to draft because when I was reviewing it I have realised that I deleted this because I thought "menu doesn't have sub nav anymore" but I realise now that I was probably mixing this with hmrcAccountMenu - I'm not going to do any further changes, I'll wait to talk to Tim when he's back because it's more work than expected to get these working